clients/py/v3io_frames/client.py

```diff
@@ -25,18 +25,22 @@
 class ClientBase:
     def __init__(self, address, session, frame_factory=pd.DataFrame,
                  concat=pd.concat):
-        """Create new client
+        """Creates a new Frames client object
 
         Parameters
         ----------
         address : str
-            framesd server address
+            Address of the Frames service (framesd)
         session : Session
             Session object
         frame_factory : class
-            DataFrame factory (currencly pandas and cudf supported)
+            DataFrame factory; currently, pandas and cuDF are supported
```

Comment:
Not sure we want to say that cuDF is officially supported yet

Reply:
I agree. @talIguaz @dinal feel free to remove the cuDF reference from the help text; note that it's not included in the README. (As I didn't write the original help text, I didn't want to take it upon myself to remove the cuDF reference from this text and I didn't have time to ask last week.)